src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadget_lib.h

```diff
@@ -356,7 +356,29 @@ namespace sirf {
 		}
 	};
 
-    class GenericReconCartesianFFTGadget : public Gadget {
+	class FFTGadget : public Gadget {
+	public:
+		FFTGadget() :
+			Gadget("FFT", "gadgetron_mricore", "FFTGadget")
+		{}
+		static const char* class_name()
+		{
+			return "FFTGadget";
+		}
+	};
+
+	class CombineGadget : public Gadget {
```

Comment:
add some doxygen what this is supposed to do

Reply:
done